#
# Fix typo in rfc2459.py
# Typo has been fixed in upstream but was not released yet.
#
--- pyasn1-modules-0.0.4/pyasn1_modules/rfc2459.py	2012-04-11 02:34:52.000000000 -0700
+++ pyasn1-modules-0.0.4/pyasn1_modules/rfc2459.py	2013-07-03 12:20:17.801945873 -0700
@@ -761,7 +761,7 @@
 id_ce_cRLDistributionPoints = univ.ObjectIdentifier('2.5.29.31')
 
 class CRLDistPointsSyntax(univ.SequenceOf):
-    componentType = DistributionPoint
+    componentType = DistributionPoint()
     subtypeSpec = univ.SequenceOf.subtypeSpec + constraint.ValueSizeConstraint(1, MAX)
 id_ce_issuingDistributionPoint = univ.ObjectIdentifier('2.5.29.28')
 
